netlify.toml

```diff
@@ -279,6 +279,11 @@
   from = "/*/eth2/docking"
   to = "/:splat/eth2/merge/"
 
+# Jobs
+[[redirects]]
+  from = "/*/about/web-developer"
+  to = "/:splat/community/#ethereum-jobs"
```

Comment:
@samajammin Do we want to immediately redirect to third-party job listings? Or should we redirect user internally first to `/:splat/about/#open-jobs` and place a snippet in here along the lines of "Not seeing the right role for you?" with a link to `/community/#ethereum-jobs`